src/util/secretUpdater.ts

```diff
@@ -0,0 +1,8 @@
+import { ExtensionContext } from "vscode";
+
+
+// Before Building, Please update secrets else IQ Feedback service will not work
+export const secretUpdater = (context: ExtensionContext) => {
+    context.globalState.update("feedbackLambdaSecret", "c0uchbase_is_aw3some");
```

Comment:
```suggestion
    context.globalState.update("feedbackLambdaSecret", "c0uchbase_is_aw3some");
```
Can you put it in environment variable?

Reply:
Keeping this as bug and priority fix